data/json/items/comestibles/veggy_dishes.json

```
@@ -1167,6 +1167,45 @@
     "delete": { "flags": [ "RAW" ] },
     "fun": 2
   },
+  {
+    "type": "COMESTIBLE",
+    "id": "rehydrated_chili",
+    "name": { "str": "rehydrated chili pepper" },
+    "weight": "48 g",
+    "color": "red",
+    "spoils_in": "10 days",
+    "comestible_type": "FOOD",
+    "symbol": "%",
+    "calories": 13,
+    "quench": -8,
+    "description": "Reconstituted chili pepper.  It isn't less spicy after dehydrating and rehydrating but it can be used to flavor your dishes again.",
+    "price": "16 cent",
+    "price_postapoc": "16 cent",
+    "material": [ "veggy" ],
+    "volume": "70 ml",
+    "flags": [ "EDIBLE_FROZEN" ],
+    "vitamins": [ [ "vitC", 88 ], [ "calcium", 1 ], [ "iron", 3 ] ],
+    "fun": -2
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "rehydrated_garlic",
+    "name": { "str_sp": "rehydrated garlic cloves" },
```

```suggestion
    "name": { "str": "rehydrated garlic clove" },
```
Consistency with your own earlier change